interface/patient_file/encounter/new_form.php

```diff
@@ -39,35 +35,35 @@ function toggleFrame1(fnum) {
 </script>
 <style type="text/css">
 #sddm
-{	margin: 0;
-	padding: 0;
-	z-index: 30;
+{   margin: 0;
+    padding: 0;
+    z-index: 30;
 }
 
 </style>
 <script type="text/javascript" language="javascript">
 
-var timeout	= 500;
-var closetimer	= 0;
-var ddmenuitem	= 0;
+var timeout = 500;
```

Comment:
Just stating that while it is changing the white-space, it isn't making it pretty.  Sorry I was bitching.  This commit is very important.